ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -123,10 +128,10 @@ def validate_config(self, config):
 
                 # Check if default groups exist
                 context = {'model': model, 'user': c.user}
-                for group_name in config_obj['default_groups']:
+                for group_ in config_obj['default_groups']:
```

Comment:
This seems wrong. `default_groups` is (or should be) a list of of strings (names or ids). Why do you assume they are a dict?